packages/core/src/routes/organization/roles.ts

```diff
@@ -57,30 +61,52 @@ export default function organizationRoleRoutes<T extends AuthedRouter>(
   /** Allows to carry an initial set of scopes for creating a new organization role. */
   type CreateOrganizationRolePayload = Omit<CreateOrganizationRole, 'id'> & {
     organizationScopeIds: string[];
+    scopeIds: string[];
   };
 
+  // TODO @wangsijie - Remove this once the feature is ready
+  const originalCreateCard: z.ZodType<
+    Omit<CreateOrganizationRolePayload, 'scopeIds'> & { scopeIds?: string[] },
+    z.ZodTypeDef,
+    unknown
+  > = OrganizationRoles.createGuard
+    .omit({
+      id: true,
+    })
+    .extend({
+      organizationScopeIds: z.array(z.string()).default([]),
+    });
+
   const createGuard: z.ZodType<CreateOrganizationRolePayload, z.ZodTypeDef, unknown> =
     OrganizationRoles.createGuard
       .omit({
         id: true,
       })
       .extend({
         organizationScopeIds: z.array(z.string()).default([]),
+        scopeIds: z.array(z.string()).default([]),
```

Comment:
```suggestion
        resourceScopeIds: z.array(z.string()).default([]),
```

align naming with other places?

Reply:
updated